homeassistant/components/nexia/binary_sensor.py

```diff
@@ -0,0 +1,114 @@
+"""Support for Nexia / Trane XL Thermostats."""
+
+from homeassistant.components.binary_sensor import BinarySensorDevice
+from homeassistant.const import ATTR_ATTRIBUTION
+
+from .const import (
+    ATTR_THERMOSTAT_ID,
+    ATTRIBUTION,
+    DATA_NEXIA,
+    DOMAIN,
+    MANUFACTURER,
+    NEXIA_DEVICE,
+    UPDATE_COORDINATOR,
+)
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
```

Comment:
For your next PR, limit it to the foundation (`__init__.py` + config flow) and 1 platform. Don't include multiple platforms in the initial PR.

Reply:
Will do